### PR TITLE
Cut text using Ctrl+x

### PR DESCRIPTION
**Pull request by RustemB**

### CODE REVIEW

File: sixtyfps_runtime/corelib/items/text.rs [hunk 1]
```diff
@@ -396,6 +396,10 @@ impl Item for TextInput {
                     } else if event.text == "v" {
                         self.paste(window);
                         return KeyEventResult::EventAccepted;
+                    } else if event.text == "x" {
+                        self.copy();
+                        self.insert("", window);
```
Comment (tronical) [inline sixtyfps_runtime/corelib/items/text.rs:401]:
```suggestion
                        self.delete_selection(window);
```

**Review by tronical — COMMENTED** [review]
Looks generally good - thanks! I made one small suggestion. Would you have time to also add a test case? (Like in tests/cases/text) otherwise I can also do that on Monday

**Comment by RustemB** [comment]
Ok, i'll do it

**Comment by RustemB** [comment]
Am I allowed to put my name to the header of a test file? :D

**Comment by tronical** [comment]
> Am I allowed to put my name to the header of a test file? :D

Sorry, our script is overzealous :) - I’ll look into fixing that.

**Review by tronical — APPROVED** [review]

**Comment by ogoffart** [comment]
Thanks for the changes!

Should there be a changelog entry?

**Comment by RustemB** [comment]
I think should, because this is important feature, as in most toolkits

**Comment by tronical** [comment]
I can add it after landing this, unless you'd like to add a commit on top of your branch with an entry :)

**Pull request merged**